ui/src/widgets/ui-radio-group/UIRadioGroup.vue

```diff
@@ -143,3 +143,9 @@ export default {
     }
 }
 </script>
+
+<style>
+.nrdb-ui-radio-group .v-radio-group>.v-input__control > .v-label {
```

Comment:
Gayan, does this need to be do specific? 

I'm worried users won't be able to style this so easily with their own CSS. 

Is there a chance the selector can be reduced to the minimum? 

It's ok if not (I am no CSS expert, this is just an observation)

Reply:
Better to make more specific as the Vuetify has used the same path to apply CSS otherwise we may have to force apply the CSS with `!important`

<img width="489" alt="Screenshot 2024-09-05 at 00 14 39" src="https://github.com/user-attachments/assets/7a733282-a6ed-4798-bee6-491eb893da8a">

The best we can do is to reduce the selector path to this `.nrdb-ui-radio-group .v-radio-group .v-label`

Reply:
Thanks gayan

Reply:
@Steve-Mcl it doesn't override when we skipped `v-input__control` class. We have to specify it but it's not necessary to catch next immediate child element using `>` operator

The updated selector is `.nrdb-ui-radio-group .v-radio-group .v-input__control .v-label` [here](https://github.com/FlowFuse/node-red-dashboard/pull/1268/commits/68a70a4d3382400245bdcac0b4d10882475625c1)